### PR TITLE
Update ReversingLabs solution

### PR DESCRIPTION
**Pull request by AaronHoffmannRL**
Required items, please complete
   
   Change(s):
   - This PR re-introduces the ReversingLabs Sentinel solution
   - Adds 1 new playbook to replace the previous playbook
   - Adds 1 new workbook

   Reason for Change(s):
   - Updating solution to better fit capabilities

   Version Updated:
   - N/A

   Testing Completed:
   - Yes

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes

### CODE REVIEW

File: Solutions/ReversingLabs/Data/Solution_ReversingLabs.json [hunk 1]
```diff
@@ -1,12 +1,23 @@
...
+  "Parsers": [],
+  "SavedSearches": [],
+  "Hunting Queries": [],
+  "Data Connectors": [],
+  "Watchlists": [],
+  "WatchlistDescription": [],
+  "BasePath": "C:\\GitHub\\Azure-Sentinel\\Solutions\\ReversingLabs",
+  "Version": "1.0.0",
+  "Metadata": "SolutionMetadata.json",
+  "TemplateSpec": false
```
Comment (v-marimanda) [inline Solutions/ReversingLabs/Data/Solution_ReversingLabs.json:22]: @AaronHoffmannRL  can please change  TemplateSpec value  true and please add one more property "Is1Pconnector": false.

File: Solutions/ReversingLabs/Data/Solution_ReversingLabs.json [hunk 2]
```diff
@@ -1,12 +1,23 @@
...
+	  "Playbooks\\ReversingLabs-EnrichFilehash\\azuredeploy.json"
+  ],
+  "Parsers": [],
+  "SavedSearches": [],
+  "Hunting Queries": [],
+  "Data Connectors": [],
+  "Watchlists": [],
+  "WatchlistDescription": [],
+  "BasePath": "C:\\GitHub\\Azure-Sentinel\\Solutions\\ReversingLabs",
+  "Version": "1.0.0",
```
Comment (v-marimanda) [inline Solutions/ReversingLabs/Data/Solution_ReversingLabs.json:20]: @AaronHoffmannRL Please update the version 2.0.0.thanks!!

**Review by v-marimanda — CHANGES_REQUESTED** [review]

**Comment by v-marimanda** [comment]
@AaronHoffmannRL Please update package with above changes as well. thanks!!!

**Comment by v-marimanda** [comment]
@rushriva  and @devikamehra  Please review the playbook provide your sign off. thanks!!!

**Comment by v-marimanda** [comment]
Workbook is looking good. 
image

**Comment by AaronHoffmannRL** [comment]
@v-marimanda requested changes to Solution_ReversingLabs.json have been completed

**Comment by v-marimanda** [comment]
@AaronHoffmannRL  Please update package with v2 tool. thanks!!!
Tool reference path : https://github.com/Azure/Azure-Sentinel/tree/master/Tools/Create-Azure-Sentinel-Solution/V2

**Comment by v-marimanda** [comment]
@AaronHoffmannRL Please don't delete old zip files from package folder. thanks!!!

**Comment by v-marimanda** [comment]
@rushriva  and @devikamehra  Please review the playbook provide your sign off. thanks!!!

**Comment by v-marimanda** [comment]
@rushriva and @devikamehra Please review the playbook provide your sign off. thanks!!!

**Comment by AaronHoffmannRL** [comment]
@v-marimanda I've re-packaged the solution with the V2 script and added the old ZIP archives back to the folder

**Comment by v-marimanda** [comment]
@manishkumar1991  Please review the playbook provide your sign off. thanks!!!

**Comment by v-marimanda** [comment]
@rahul0216  and @manishkumar1991  Please review the playbook provide your sign off. thanks!!!

**Review by manishkumar1991 — APPROVED** [review]
All changes Looks good , 
Createuidefinition file and maintemplate files are also giving successful results

**Comment by v-atulyadav** [comment]
Hi @manishkumar1991 , can you please merge this PR. Thanks.

**Comment by v-atulyadav** [comment]
Hi @manishkumar1991 , can you please merge this PR. Thanks.

**Review by v-marimanda — APPROVED** [review]

**Pull request merged**